src/bun.js/streams.exports.js

```diff
@@ -4674,46 +5347,49 @@ var require_duplex = __commonJS({
     module.exports = Duplex;
 
     {
-      const keys = ObjectKeys(Writable.prototype);
+      const keys = ObjectKeys(WritableReadable.prototype);
       for (let i = 0; i < keys.length; i++) {
         const method = keys[i];
         if (!Duplex.prototype[method])
```

Comment:
This calls the getter - we don't want to do that

Instead, let's do `method in prototype`